pyan/analyzer.py

```diff
@@ -407,9 +425,9 @@ def visit_ImportFrom(self, node):
                 self.set_value(new_name, tgt_id)
                 self.logger.info("From setting name %s to %s" % (new_name, tgt_id))
 
-        else:  # module name missing = "from . import ..."
-            for import_item in node.names:  # the names are modules
-                self.analyze_module_import(import_item, node)
+        #else:  # module name missing = "from . import ..."
+        #    for import_item in node.names:  # the names are modules
+        #        self.analyze_module_import(import_item, node)
```

Comment:
Left over from testing? As a general rule, commented-out code should be removed, since git itself already stores historical revisions.